packages/reactive_ble_mobile/protos/bledata.proto

```diff
@@ -2,6 +2,16 @@ syntax = "proto3";
 option java_package = "com.signify.hue.flutterreactiveble";
 option java_outer_classname = "ProtobufModel";
 
+message LaunchCompanionRequest {
+    string pattern = 1;
```

Comment:
I think we should call it everywhere deviceNamePattern what do you think? Else people can be confused what it is.

- [ ]  Change to `deviceNamePattern `